src/software/jetson_nano/services/motor.cpp

```diff
@@ -387,46 +387,19 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
         current_euclidean_velocity[2]);
 
     WheelSpace_t target_wheel_velocities = {0.0, 0.0, 0.0, 0.0};
-
-    EuclideanSpace_t target_linear_velocity  = {0.0, 0.0, 0.0};
-    EuclideanSpace_t target_angular_velocity = {0.0, 0.0, 0.0};
-    int target_dribbler_rpm                  = motor.dribbler_speed_rpm();
-    static int ramp_rpm                      = 0;
-
-    switch (motor.drive_control_case())
-    {
-        case TbotsProto::MotorControl::DriveControlCase::kDirectPerWheelControl:
-        {
-            target_wheel_velocities = {
-                motor.direct_per_wheel_control().front_right_wheel_velocity(),
-                motor.direct_per_wheel_control().front_left_wheel_velocity(),
-                motor.direct_per_wheel_control().back_left_wheel_velocity(),
-                motor.direct_per_wheel_control().back_right_wheel_velocity()};
-
-            break;
-        }
-        case TbotsProto::MotorControl::DriveControlCase::kDirectVelocityControl:
-        {
-            target_linear_velocity = {
-                -motor.direct_velocity_control().velocity().y_component_meters(),
-                motor.direct_velocity_control().velocity().x_component_meters(),
-                motor.direct_velocity_control().angular_velocity().radians_per_second()};
-        };
-
-        break;
-        case TbotsProto::MotorControl::DriveControlCase::DRIVE_CONTROL_NOT_SET:
-        {
-            target_linear_velocity  = {0.0, 0.0, 0.0};
-            target_angular_velocity = {0.0, 0.0, 0.0};
-            target_dribbler_rpm     = 0;
-
-            break;
-        }
-    }
-    target_wheel_velocities = rampWheelVelocity(
-        prev_wheel_velocities, target_linear_velocity,
-        static_cast<double>(robot_constants_.robot_max_speed_m_per_s),
-        static_cast<double>(robot_constants_.robot_max_acceleration_m_per_s_2),
+    int target_dribbler_rpm =
+        motor.drive_control_case() ==
+                TbotsProto::MotorControl::DriveControlCase::DRIVE_CONTROL_NOT_SET
+            ? 0
+            : motor.dribbler_speed_rpm();
+    static int ramp_rpm = 0;
```

Comment:
Can we make this a member variable (not static and placed in the the header file. This is a technical debt we've had for a while.